src/pybind/mgr/devicehealth/module.py

```diff
@@ -215,15 +215,15 @@ def do_get_health_metrics(self, devid: str, sample: Optional[str] = None) -> Tup
         return self.show_device_metrics(devid, sample)
 
     @CLIRequiresDB
-    @CLICommand('device check-health',
+    @CLIWriteCommand('device check-health',
                 perm='rw')
```

Comment:
This change is wrong. See `CLIWriteCommand` is for write only command https://github.com/ceph/ceph/blob/84be9581dc0ac9f064d5095708eb5d15d506178d/src/pybind/mgr/mgr_module.py#L433-L434

Reply:
This needs to be
```suggestion
    @CLICommand('device check-health')
```

Reply:
Okay